utils/parse.js

```diff
@@ -128,6 +128,13 @@ module.exports.result = function parseResult(html, year, callback) {
     result.anneeImpots = regexp.exec(titleAnnee)[0];
     result.anneeRevenus = regexp.exec(titleAnnee)[0];
   }
+
+  var nodeErreurCorrectif = select('//*[@id="erreurCorrectif"]', doc);
+  if (nodeErreurCorrectif.length > 0) {
+    var erreurCorrectif = nodeErreurCorrectif[0].firstChild.data + ' ' + nodeErreurCorrectif[0].lastChild.data;
```

Comment:
Sommes nous sûr que le node erreurCorrectif ne contienne que 2 childs ?

Sinon, je recommande une implémentation plus robuste : l'utilisation de childNodes avec une boucle d'itération (cf. https://stackoverflow.com/a/30751448/2590861).

```js
if (nodeErreurCorrectif.length > 0) {
  var erreurCorrectif = nodeErreurCorrectif[0].firstChild.data;
  for (var i = 1; i < nodeErreurCorrectif[0].childNodes.length; i++) {
    erreurCorrectif += ' ' + nodeErreurCorrectif[0].childNodes[i].data;
  }
}
```

Reply:
attention, le code que j'ai fourni n'a pas été testé. Si c'est trop compliqué de le faire fonctionner on pourra rester sur votre version initiale et améliorer le code dans un second temps.

Reply:
je vous propose une implémentation que je pense plus robuste.
J'ai aussi intégré le traitement pour la situation partielle.
Et enfin je vous propose 2 ressources pour les tests qui restent encore à écrire.

Merci !